yt_dlp/extractor/eporner.py

```diff
@@ -106,6 +110,14 @@ def calc_hash(s):
                         'height': height,
                         'fps': fps,
                     })
+                    if av1:
+                        formats.append({
+                            'url': re.sub('(.*)(.mp4)', '\\1-av1\\2', src),
+                            'format_id': join_nonempty('AV1', format_id),
```

Comment:
I'd just use a simple `str.replace` instead of the regex. If using regex though, always use raw strings for patterns
```suggestion
                    if has_av1:
                        formats.append({
                            'url': src.replace('.mp4', '-av1.mp4'),
                            'format_id': join_nonempty('av1', format_id),
```